src/BeanstalkClient.php

```diff
@@ -294,24 +318,59 @@ public function getSystemStats(): Promise {
 
             switch ($type) {
                 case "OK":
-                    return new System($this->getStatsFromString($response[1]));
+                    return new System(Yaml::parse($response[1]));
 
                 default:
                     throw new BeanstalkException("Unknown response: " . $type);
             }
         });
     }
 
-    private function getStatsFromString(string $stats): array {
-        $result = [];
-        $source = explode("\n", $stats);
-        foreach ($source as $stat) {
-            if ($stat == '---' || empty($stat)) {
-                continue;
+    public function getTubesList(): Promise {
```

Comment:
We could name it `listTubes()`.

Reply:
I'm not completely sure about it. For me, its more predictable to have a `getTubesList()` and `getWatchedTubesList()` methods. What do you think ?

Reply:
I think `listWatchedTubes()` reads slightly better, but I don't have a strong preference for either.

@markkimsal Thoughts?